workflow/resources/datavzrd/go-enrichment-template.yaml

```diff
@@ -1,105 +1,232 @@
+  #         display-mode: normal
+  #       class:
+  #         plot:
+  #           heatmap:
+  #             scale: ordinal
+  #       p-value:
+  #         plot:
+  #           bars:
+  #             scale: linear
+  #             domain:
+  #               - 0.0
+  #               - 1.0
+  #       p_corrected:
+  #         plot:
+  #           bars:
+  #             scale: linear
+  #             domain:
+  #               - 0.0
+  #               - 1.0
+  #       n_genes_diff_exp:
+  #         plot:
+  #           heatmap:
+  #             scale: linear
+  #       n_genes_diff_exp_study:
+  #         display-mode: normal
+  #       n_go_genes:
+  #         plot:
+  #           bars:
+  #             scale: linear
+  #       gene_ratio:
+  #         plot:
+  #           heatmap:
+  #             scale: linear
+  #       study_items:
+  #         display-mode: normal
   go_enrichment:
     dataset: go_enrichment
     desc: |
-        Gene ontology (GO) term enrichment analysis performed by goatools, using the sleuth model. The table contains the following columns. "GO" is the GO ID, "term" is the term associated with the GO, "class" is the corresponding class to which the GO term belongs (biological process/cellular component/molecular function), "p_uncorrected" is the p-value obtained, "p_fdr_bh" is the p value obtained from false discovery, "ratio_in_study" is the ratio of differentially expressed genes in the current study, "ratio_in_pop" is the ratio of total genes present in the current analysis, "depth" is the total number of GO terms associated with the current ID, "study_count" is the number of differentially expressed genes in that particular GO term, "study_items" is the name of each differentially expressed genes. Each gene contains its corresponding "b" value with is the fold change seperated by ":". "Gene ratio" gives the ratio of gene corresponding to each term.  
+      <div">
+        <p>
+          Gene ontology (GO) term enrichment analysis performed by goatools, using the sleuth model. 
+          The table contains the following columns:
+        </p>
+        <div>
+          <table border="1" cellpadding="5" cellspacing="0">
+              <tr>
+                  <td><strong>Column Name</strong></th>
+                  <td><strong>Description</strong></th>
+              </tr>
+              <tr>
+                  <td>term</td>
+                  <td>Term associated with the GO</td>
+              </tr>
+              <tr>
+                  <td>class</td>
+                  <td>GO term class</td>
+              </tr>
+              <tr>
+                  <td>ratio_differential</td>
+                  <td>Ratio of differentially expressed genes in the GO term</td>
+              </tr>
+              <tr>
+                  <td>ratio_size</td>
+                  <td> Ratio of GO term size in relation to total number of measured genes</td>
+              </tr>
+              <tr>
+                  <td>study_items</td>
```

Comment:
Having HTML here seems ugly. Just use markdown. Also, it does not need to be a table. Prose text is even better, as one can also paste that in the caption of a paper.